src/main/java/com/fryrank/dal/ReviewDALImpl.java

```diff
@@ -50,6 +51,16 @@ public GetAllReviewsOutput getAllReviewsByAccountId(@NonNull final String accoun
         return new GetAllReviewsOutput(reviews);
     }
 
+    @Override
+    public GetAllReviewsOutput getTop10MostRecentReviews(){
+        final Query query= new Query();
+        query.with(Sort.by(Sort.Direction.DESC, "isoDateTime"));
```

Comment:
Making sure that `DESC` will return reviews in reverse-chronological order (i.e. most recent first) which is the convention we follow for displaying reviews on the website